localization/english/CWP/CWP_resolutions_l_english.yml

```diff
@@ -26,8 +26,20 @@
   concept_un_headquarters_resolution: "UN Headquarters Resolution"
   concept_decolonization_resolution: "Convention on Decolonization"
   concept_apartheid_resolution: "Convention on Apartheid"
-  concept_environmental_modification_resolution: "ENMOD"
+  concept_environmental_modification_resolution: "ENMOD""
   concept_sustainable_development_resolution: "Sustainable Development"
   concept_unaec_resolution: "UNAEC"
   concept_unga_partition_resolution: "UNGA Partition Resolution"
   concept_resolution_on_korea: "Resolution on Korea"
+  
```

Comment:
error with this line and below. all yaml lines must be indented to the same level. The indentation of this file skips from 1 space to 0 space. See note below.